nload: add page (#5395)
tldr-pages/tldr: pages/common/nload.md

# nload

> A tool for visualizing network usage in the terminal.
> More information: <https://github.com/rolandriegel/nload>.

> - View all network traffic (arrow keys to switch interfaces):

Reviewer: ```suggestion
- View all network traffic (use the arrow keys to switch interfaces):
```

`nload`

> - View network traffic on a particular interfaces (arrow keys to switch interfaces):

Reviewer: ```suggestion
- View network traffic on a particular interfaces (use the arrow keys to switch interfaces):
```

> - View network traffic on a particular interfaces (use the arrow keys to switch interfaces):

Reviewer 2: ```suggestion
- View the network traffic on specific interfaces (use the arrow keys to switch interfaces):
```

Reviewer 2: Sure. Is this a general rule or just in this example?

Reviewer: Just in this example 👍🏻

Reviewer: If you feel strongly otherwise, we don't need at all.
